distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -30,6 +30,9 @@ javax.net.ssl.trustStore=etc/keystores/serverTruststore.jks
 javax.net.ssl.trustStorePassword=changeit
 javax.net.ssl.keyStoreType=jks
 
```

Comment:
We should refer to the doc here so people know what this system property will impact.